time/src/main/java/net/jqwik/time/api/arbitraries/InstantArbitrary.java

```diff
@@ -0,0 +1,128 @@
+package net.jqwik.time.api.arbitraries;
+
+import java.time.*;
+import java.time.temporal.*;
+
+import org.apiguardian.api.*;
+
+import net.jqwik.api.*;
+
+import static org.apiguardian.api.API.Status.*;
+
+/**
+ * Fluent interface to configure the generation of instant values.
+ * All generated instants use the Gregorian Calendar, even if they are before October 15, 1582.
+ * By default, instants with years between 1900 and 2500 are generated.
+ * Max possible year is 999_999_999.
+ */
+@API(status = EXPERIMENTAL, since = "1.5.5")
```

Comment:
Next version is 1.5.4. Should be doable is you don't plan for more functionality in this PR.